faq.md

```diff
@@ -0,0 +1,58 @@
+
+==- How do you hedge collateral?
+The protocol swaps the collateral to the borrower’s chosen liability token (typically stablecoins), mitigating the protocol’s exposure to any decline in the asset’s price.
+
+==- But what if the asset appreciates in value?
+Providers post stablecoins upfront to guarantee the borrower’s upside exposure to the cap.
+
+==- Why should providers use Votre?
+Because Votres are volatile bets and providers can monetize this volatility. They do this by offering terms to the user that contain a small spread, then monetizing this spread over the life of the trade. This technique is called “gamma scalping” and you can learn more about it here.
+
+==- How do I know I’m getting the best price?
+Votre allows for open competition between providers onchain to provide the best offer. With every Votre request via the hosted frontend, users are shown competitive quotes before they can execute in order to promote best execution.
+
+==- How big is this “spread”?
+Like any spread, it is a function of the number of providers. Votre will go live with three sophisticated options providers from day one who we have partnered with, and they will compete for user flow.
+
+==- Why would I cap my returns?
+Because in bull markets it can be beneficial to take chips off the table or even redeploy borrowed collateral into a new position for greater capital efficiency. Users can also “roll” their positions higher as assets appreciate to maintain exposure so long as there is sufficient provider-provided liquidity.
+
+==- How is the final price determined?
+Uniswap TWAP oracles
+
+==- What venues does the protocol swap on?
+Uniswap for now, but this may expand in the future as liquidity improves
+
+==- Does swapping create slippage?
+Yes, but compared to interest rates and capital requirements of other protocols, this is minimal.
+
+==- Does Votre charge interest?
+Since users are providing the collateral required for the directional hedge, there is no need to charge interest to source collateral or funds.
+
+==- What if a provider fails like in 2022?
+All provider collateral is locked into Votre’s smart contracts, so they'll only get money back at maturity once the final price of the asset is in. Additionally, both sides of a Votre position are represented as NFTs, which can be sold, traded, or united to unwind a position.
+
+==- How is Votre different from AAVE, Compound, and other lending protocols?
+Other protocols liquidate your collateral at the worst possible time. Votre creates a time-based, as opposed to a price-based loan.
+
+==- Does swapping create a taxable event?
+Protocols can’t give tax advice, but this is an issue that is solved by a feature called “supplier mode”, which introduces suppliers to Votre. By escrowing user collateral day one and charging users interest, Votres may allow capital gains to be deferred in some jurisdictions. Speak to your tax professional for more information.
```

Comment:
Would replace the part with "Depends on the borrower's jurisdiction's tax laws.."